bundle/manifests/nvidia-network-operator.clusterserviceversion.yaml

```diff
@@ -693,12 +704,3 @@ spec:
     targetPort: 9443
     type: ValidatingAdmissionWebhook
     webhookPath: /validate-mellanox-com-v1alpha1-nicclusterpolicy
-  relatedImages:
```

Comment:
@e0ne should we think about a hack to populate `relatedImages` automatically?

Reply:
we need to see how to add those when generating bundle

sounds like there should be some "official" way to do it.

@killianmuldoon  can you add this section back meanwhile ? otherwise LGTM

Reply:
there is also: https://github.com/operator-framework/operator-sdk/issues/4884

Reply:
created: https://github.com/Mellanox/network-operator/issues/765

Reply:
I can add them manually, but I'm not sure what the correct versions are. Are these the shas of the repos at specific tagged versions?

Reply:
Here are the latest versions published in OpenShift:

https://github.com/redhat-openshift-ecosystem/certified-operators/pull/3199/files#diff-8aa18b74f844492136b4fb0d11995eccdfe22be79e7023d40363140fcb46abf9R664

Reply:
Should be up to date now - thanks for catching this one!